src/options/pages/services/ServiceEditorModal.module.scss

```
@@ -24,3 +24,9 @@
   // Modal is 980
   z-index: 980 !important;
 }
+
+.dialog {
+  width: 55vw;
+  max-width: 700px;
+  min-width: 400px;
```

This is probably what we should aim for:

```suggestion
.dialog {
  width: 100%; // maybe not needed at all
  max-width: 700px;
```

I think it never makes sense to target a _percentage of the width_ when showing a single item on screen.